site/_css/algolia.less

```diff
@@ -27,7 +27,7 @@
     transition: width 0.2s ease-out;
 
     &:focus {
-      width: 300px;
+      width: 250px;
```

Comment:
This much difference will then move it to the bottom on a smaller screen resolution. This may work on your screen size but will not in many others. This is not a permanent fix or solution.

Reply:
Actually we need a sidebar rather than navbar for smaller devices.